libs/payments/ui/src/lib/client/components/StripeWrapper.tsx

```diff
@@ -153,3 +153,64 @@ export function StripeWrapper({
     </Elements>
   );
 }
+
+interface StripeManagementWrapperProps {
+  locale: string;
+  currency: string;
+  sessionSecret: string;
+  children: React.ReactNode;
+  instanceKey: string;
+}
+
+export function StripeManagementWrapper({
+  locale,
+  currency,
+  sessionSecret,
+  children,
+  instanceKey,
+}: StripeManagementWrapperProps) {
+  const config = useContext(ConfigContext);
+  const [stripePromise] = useState(() => loadStripe(config.stripePublicApiKey));
+
+  const options: StripeElementsOptions = {
```

Comment:
suggestion: Could the options common to `StripeWrapper` and `StripeManagementWrapper` be moved to a separate variable that can be used by both components?